docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -345,6 +345,21 @@ strapi generate:plugin <name>
 
 Please refer to the [local plugins](/developer-docs/latest/development/local-plugins-customization.md) section to know more.
 
+## strapi generate:template
+
+Create a template from the current strapi project
+
+```bash
+strapi generate:template <path>
+```
+
+- **strapi generate:template &#60;path&#62;**<br/>
```

Comment:
Could you please indent this so that it looks like the other sections? 
See the "generate:plugin" section above as an example. 
Thank you
![Screenshot 2021-06-04 at 14 49 14](https://user-images.githubusercontent.com/4233866/120803884-50dc6080-c544-11eb-94f3-aa71ddfcc3e5.png)